docs/integrations/llms.md

```diff
@@ -0,0 +1,14 @@
+The Pydantic documentation is available in the [llms.txt](https://llmstxt.org/) format.
+This format is defined in Markdown and suited for large language models.
+
+Two formats are available:
+
+- [llms.txt](https://docs.pydantic.dev/latest/llms.txt): a file containing a brief description
```

Comment:
We could use https://github.com/fralau/mkdocs-macros-plugin to dynamically build the URL from `config.site_url`, but it seems that this plugin treats every markdown page as a jinja template, and I'm worried this sill lead to unexpected side effects